RAII_Samples/utils/CMakeLists.txt

```diff
@@ -27,11 +27,11 @@ set(SOURCES
 source_group(headers FILES ${HEADERS})
 source_group(sources FILES ${SOURCES})
 
-add_library(RAII_utils
+add_library(RAII_utils INTERFACE
```

Comment:
MacOS linker wants to have at lease one source file to create a `*.a` target. This is an interface anyway